modules/apps/static/portal-osgi-web/portal-osgi-web-portlet-tracker/src/main/java/com/liferay/portal/osgi/web/portlet/tracker/internal/PortletTracker.java

```diff
@@ -176,7 +182,7 @@ public com.liferay.portal.kernel.model.Portlet addingService(
 			_log.info("Adding " + serviceReference);
 		}
 
-		portletModel = _addingPortlet(
+		com.liferay.portal.kernel.model.Portlet portletModel = _addingPortlet(
```

Comment:
iinm, this code can now be reached with the same `portletId` right? It's kinda surprising to me that we don't we need any special treatment in the `_addingPortlet()` method

Reply:
> It's kinda surprising to me that we don't we need any special treatment in the _addingPortlet() method

Not sure I understand the suggestion... :thinking: 

What type of treatment were you expecting?